yamllint/rules/key_ordering.py

```diff
@@ -78,8 +93,17 @@
       haïr: true
       hais: true
       haïssable: true
-"""
 
+#. With rule ``key-ordering: {ignored-keys: ["name"]}``
+
+   the following code snippet would **PASS**:
+   ::
+
+    - name: John
```

Comment:
From this example, it gives the _appearance_ of string matches. If this was something besides `name` or the sample input had regex chars, it would make the relationship more clear.

```suggestion
    - first-name: John
```